addon/components/dropzone-widget/component.js

```diff
@@ -80,6 +80,14 @@ export default Ember.Component.extend({
             }
         });
 
+        // Dropzone.js does not have an option for disabling selecting multiple files when clicking the "upload" button.
+        // Therefore, we remove the "multiple" attribute for the hidden file input element, so that users cannot select
+        // multiple files for upload in the first place.
+        let clickableElement = this.get('clickable');
+        if (this.options && this.options.preventMultipleFiles && clickableElement){
```

Comment:
Should use this.get() e.g. `this.get('options.preventMultipleFiles')`

Reply:
`if (this.get('options.preventMultipleFiles') && this.get('clickableElement')) {`